README.md

```diff
@@ -9,61 +9,109 @@ An Architecture Decision Record (ADR) is a document that captures an important a
 ## Introduction
 
 * [Why write architecture decision records - By GitHub Engineering](https://github.blog/engineering/architecture-optimization/why-write-adrs/)
+* [A practical overview on Architecture Decision Records](https://ctaverna.github.io/adr/)
 
-* [A practical overview on Architecture Decision Records: How to start and why this could be your most valuable action as a software architect](https://ctaverna.github.io/adr/)
+## Getting Started
 
-## Documentation
+There are several ways to set up your development environment:
 
-Our records are published using [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/).
+### 1. GitHub Codespaces (Recommended)
 
-### Running via a Container (Podman / Docker)
```

Comment:
Note to self: add instruction for building and serving mkdocs site from inside the dev container here.